.travis.yml

```diff
@@ -2,6 +2,8 @@ language: php
 php:
   - 5.5
   - 5.6
+  - 7.0
+  - hhvm
```

Comment:
I don't see any test results produced by hhvm jobs, here is the example of such: https://travis-ci.org/Sanuch/magento2/builds/47961569

You can take needed changes from @Sanuch fork or remove hhvm occurrences from this PR.